docs/core/entity/sensor.md

```diff
@@ -15,6 +15,7 @@ Properties should always only return information from memory and not do I/O (lik
 | ---- | ---- | ------- | -----------
 | state | string | **Required** | The value of the sensor.
 | device_class | string | `None` | Type of sensor.
+| last_reset | `datetime.datetime` | `None` | The time when an accumulating sensor such as an electricity usage meter, gas meter, water meter etc. was initialized. If the time of initialization is unknown and the meter will never reset, set to UNIX epoch 0: `datetime.fromtimestamp(0)`. When changing `last_reset`, the `state` must be a valid number.
```

Comment:
Did we change this to an isoformatted datetime string?

Reply:
Yes and no, the property should return a `datetime.datetime`, which will be converted to an ISO 8601-converted string by `SensorEntity.state_attributes()`.

Reply:
Ok.